src/proptests.rs

```diff
@@ -0,0 +1,61 @@
+use proptest::test_runner::{TestRunner};
+use proptest::strategy::{Strategy, ValueTree};
+use proptest::prelude::*;
+
```

Comment:
I feel like I am not using the syntax in the right way, feedback would be appreciated.